[FIX] Fix test ERROR: odoo.addons.base.tests.test_views: Please indic…

Georgeals opened the pull request:
…ate why the always invisible fields are present in the view, or remove the field tag.

Starting with Odoo version 17 this is no longer required.

alienyst commented:
Hello, sorry for the late response.

Thank you.
I used that field to display the name of the uploaded file.
But I found the code `<field name="report_docx_template_filename" invisible="1" readonly="state != 'draft'"/>` should actually be `<field name="report_docx_template_filename" invisible="1" readonly="state != 'draft'"/>` because it is always invisible.

Georgeals commented:
You are right :)

I've added new commits and used a different approach: I added a comment <!-- Used as the file name --> to indicate why the field is present.

This is only needed to pass the testing.

alienyst commented:
Thank you, @Georgeals ! Already merged 👍
